libtenzir/builtins/operators/export.cpp

```diff
@@ -192,7 +192,7 @@ struct bridge_state {
     }
     // Live and unpersisted events we still need to filter.
     if (source != event_source::retro) {
-      const auto* bound_expr = bind_expr(slice.schema(), expr);
+      const auto* bound_expr = bind_expr(slice.schema(), normalize(expr));
```

Comment:
Can we instead normalize and validate once when creating the bridge actor instead of doing this again for every batch?